packages/bot/src/transformers/component.ts

```diff
@@ -29,6 +38,16 @@ export function transformComponent(bot: Bot, payload: DiscordMessageComponent):
     case MessageComponentTypes.SelectMenuUsersAndRoles:
       component = transformSelectMenuComponent(bot, payload as DiscordSelectMenuComponent)
       break
+    case MessageComponentTypes.Section:
+      component = transformSectionComponent(bot, payload as DiscordSectionComponent)
+      break
+    case MessageComponentTypes.File:
```

Comment:
Type of `component.file` is [unfurled media item](https://discord.com/developers/docs/components/reference#unfurled-media-item-structure), which has snake case and null, so do we need to transform it?